add.py

```diff
@@ -2,21 +2,24 @@
 import sys
 
 import pymysql as mdb
+import sqlite3
 
 from tt import *
 
 
 def run(args_dict):
     args_dict = update_args(args_dict)
 
-    db = mdb.connect(user='{}'.format(USERNAME), host=args_dict['host'],
-                     password='{}'.format(PASSWORD), db=args_dict['db'],
-                     charset='utf8', autocommit=True)
+    if args_dict['dbengine'] == 'mysql':
+        db = mdb.connect(user='{}'.format(USERNAME), host=args_dict['host'],
+                         password='{}'.format(PASSWORD), db=args_dict['db'],
+                         charset='utf8', autocommit=True)
+    elif args_dict['dbengine'] == 'sqlite':
+        db = sqlite3.connect('{}.db'.format(args_dict['db']), isolation_level=None)
```

Comment:
Let's close this out with:
```
else:
     sys.exit('You\'re trying to use a DB engine that's not currently supported. Try again, specifying `mysql` or `sqlite`.')
```

using line breaks if need be to keep the line under 100 characters long.